js/components/thumbnail.js

```diff
@@ -15,35 +14,25 @@ var Thumbnail = React.createClass({
     var updateHoverPositon = nextProps.hoverPosition != this.props.hoverPosition;
     var updateFullscreen  = nextProps.fullscreen != this.props.fullscreen && this.props.videoVr;
     var updateVrViewDirection = nextProps.vrViewingDirection != this.props.vrViewingDirection;
-    return (updateHoverPositon || updateFullscreen || updateVrViewDirection);
+    var updateImageWidth = nextProps.imageWidth !== this.props.imageWidth;
+    return (updateHoverPositon || updateFullscreen || updateVrViewDirection || updateImageWidth);
   },
 
   componentWillUnmount: function() {
     this.props.onRef(undefined);
   },
 
   render: function() {
-    var thumbnail = Utils.findThumbnail(this.props.thumbnails, this.props.hoverTime, this.props.duration, this.props.videoVr);
```

Comment:
Is this code no longer needed? I can't find where this logic is being replaced.

Reply:
It was used for thumbnailStyle. Now thumbnailStyle is props from thumnailContainer.